### PR TITLE
 :sparkles: Added a method to submit comments on the front end to dis…

### PR DESCRIPTION
**Pull request by MathieuREY4**
…play them on recipes with the username, content, and date. Also updated the CSS for better styling.

### CODE REVIEW

**Review by PixelKata — APPROVED** [review]
c'est cool ! 💯

**Review by anaisdufourneau — APPROVED** [review]
👍🏻 top

**Review by Ewan-Senergous — APPROVED** [review]
Fait gaffe au useCallback, peut-etre faire un loader.

File: client/src/pages/RecipesInstruction.jsx [hunk 1]
```diff
@@ -15,8 +15,23 @@ import heartRed from "../assets/images/heart-red.svg";
 export default function RecipesInstruction() {
   const recipe = useLoaderData();
   const [comment, setComment] = useState("");
+  const [comments, setComments] = useState([]);
+  console.info(comments);
   const stars = [1, 2, 3, 4, 5];
 
+  const fetchComments = useCallback(() => {
```
Comment (kpeset) [inline client/src/pages/RecipesInstruction.jsx]: Pourquoi utiliser useCallback ? :)

**Review by kpeset — APPROVED** [review]

**Pull request merged**